### DISCUSSION TITLE
Bar chart with a table directly underneath it

### DISCUSSION THREAD

**Discussion by Monstersolo** [apache/superset#24956] [Q&A / Help] [answered]
Hello I'm new to Apache Superset and wanted to ask if it is possible to create a bar chart where a table is directly underneath it.

The idea is that it should look something like this picture:

![image](https://github.com/apache/superset/assets/142010498/33a1ef14-57f6-4aa4-8ce0-56c7fd9796c3)

Thank you in advance!

**Comment by mattitoo** [comment, marked as answer]
This is not possible out of the box; one would need to create a new visualization or maybe add an "show data table" option to certain charts. This requires coding.

**Reply by Monstersolo** [reply]
Okay thank you
